delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/ResourceF.scala

```diff
@@ -84,9 +88,44 @@ object ResourceF {
       r.void.asJsonObject deepMerge r.value.asJsonObject
     }
 
+  private def resourceFUnitJsonLdEncoder(
+      valueContext: ContextValue
```

Comment:
What is this `ContextValue`? This is not always necessary, nor useful. If the `JsonLdEncoder[ResourceF[Unit]]` is implemented correctly, why you need this here?

Reply:
It is just a private method used to create the encoder from other methods